Plugins/Wox.Plugin.Program/Programs.cs

```diff
@@ -55,11 +56,43 @@ public List<Result> Query(Query query)
                             return true;
                         },
                         IcoPath = "Images/cmd.png"
+                    },
+                    new Result()
+                    {
+                        Title = "Open Containing Folder",
+                        Action = _ =>
+                        {
+                            context.API.HideApp();
+                            context.API.ShellRun(c.ExecutePath,true);
```

Comment:
I think opening folder shouldn't require administrator permission, if you set true here, this will show UAC dialog.